context/configSchema.js

```diff
@@ -149,4 +149,5 @@ module.exports = validator.compile({
   },
   playroomPort: { type: 'number', optional: true },
   playroomTitle: { type: 'string', optional: true },
+  isYarn: { type: 'boolean', optional: true },
```

Comment:
This isn't required as it's not publically configurable.

Reply:
Agreed. Moved from context to standalone script.